frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import cn from 'classnames';
+import { Button, Input, Label } from '../FormElements/FormElements';
+import styles from './ridermodal.module.css';
+import { ObjectType } from '../../types/index';
+
+type ModalFormProps = {
+  onSubmit: (data: ObjectType) => void;
+  formData?: ObjectType;
+}
+
+const RiderModalInfo = ({ onSubmit }: ModalFormProps) => {
+  const { register, handleSubmit } = useForm();
+  const beforeSubmit = ({ name, netid, email, phoneNumber, needs,
+    address, start, end }: ObjectType) => {
+    const startDate = new Date(`${start}`).toISOString();
+    const endDate = new Date(`${end}`).toISOString();
+    const splitName = name.split(' ');
+    const firstName = splitName[0];
```

Comment:
It might be more clear to have separate first/last name fields on the form so that they don't have to be split, and in case there is no space or more than 1 space

Reply:
Yeah, I just followed what the current designs are, but we can talk to design